src/core/Val.ml

```diff
@@ -1208,6 +1201,9 @@ struct
             let phi = I.equate ri r'i in
             let phi0 = I.cmp (I.equate (`Atom x) `Dim0) phi in
             let yi, el = Abs.unleash absi in
+
+            (* If it helps, this right here is the source of the error that appears in omega1s1. - JMS *)
+            (* Format.eprintf "Here's a bad argument to rigid_vproj: %a@.@." pp_value el; *)
```

Comment:
@favonia I don't know if this helps, but this is the call to `rigid_vproj` that causes the last example in omega1s1 to fail.